src/components/navigation/navigation.tsx

```diff
@@ -0,0 +1,35 @@
+import { UrlObject } from "url";
+import { ReactNode } from "react";
+import Link from "next/link";
+
+import { AiOutlineHeart, AiOutlineSearch, AiOutlineShoppingCart } from "react-icons/ai";
+
+export const Navigation = () => {
+  return (
+    <nav className="fixed bottom-0 z-50 w-full sm:sticky sm:w-auto sm:self-start">
+      <div className="hidden sm:block  sm:px-3 sm:py-2">ロゴ</div>
+      <div className="flex cursor-pointer gap-1 border-t sm:flex-col sm:border-none">
+        <NavigationItem label="さがす" icon={<AiOutlineSearch />} href="/" />
+        <NavigationItem label="お気に入り" icon={<AiOutlineHeart />} href="/" />
+        <NavigationItem label="お買い物" icon={<AiOutlineShoppingCart />} href="/" />
+      </div>
+    </nav>
+  );
+};
+
+type NavigationItemProps = {
+  label: string;
+  icon: ReactNode;
+  href: UrlObject | __next_route_internal_types__.RouteImpl<string>;
```

Comment:
色々やり方はありそうですが、以下の様な感じはどうでしょうか。
```suggestion
  href: ComponentProps<typeof Link>["href"];
```